src/main/webapp/app/core/websocket/websocket.service.ts

```diff
@@ -441,11 +428,4 @@ export class WebsocketService implements IWebsocketService, OnDestroy {
             return response as T;
         }
     }
-
-    // see https://stackoverflow.com/a/35651029/3802758
-    private getSessionId(): string {
-        const url = this.socket?._transport?.url;
-        const match = url?.match('.*\\/websocket\\/\\d*\\/(.*)\\/websocket.*');
-        return match ? match[1] : 'unsubscribed';
-    }
```

Comment:
Considering the error message 

```
ERROR
message:There already is a subscription for\c sub-1742063478734-713. Either use unique subscription IDs or do not create multiple subscriptions for the same destination
content-length:0
```

Removing `getSessionId()` might be problematic? 🤔

Reply:
I think this issue is because of some kind of misconfiguration of TS1

Reply:
Resolved with https://github.com/ls1intum/Artemis/pull/10392/commits/493b13cb5125cb1facda6c8398a6e48e9e7ea9f1